src/components/HeatMap/Cell/Cell.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Wrapper } from './Cell.styles';
+
+
+export default function Cell({
+  itemsNumber, row, column, activeCell, getActiveCell,
+}) {
+  const [isActive, setActive] = useState(false);
+
+  useEffect(() => {
```

Comment:
You don't need this useEffect and state since it's basically just transforming a prop. I actually wrote a [blog post](https://jkettmann.com/dont-useeffect-as-callback/) about this since I saw it a couple of times

Reply:
I think I don't see this. I need to compare on "componentdidmount" for each cell if the activecell stored in the heatmap state and passed down match the actual cell...

Reply:
Yeah, but you can simply use a const for that

```
export default function Cell({ ... }) {
  const isActive = activeCell[0] === row && activeCell[1] === column;

  return ...
}
```

This way `isActive` is still always up to date, but you neither need the state nor the useEffect

Reply:
oh yeah, it works :)